terraform/modules/paas/main.tf

```diff
@@ -34,6 +34,25 @@ resource cloudfoundry_app web_app {
   }
 }
 
+
+resource cloudfoundry_app worker_app {
+  name              = local.worker_app_name
+  command           = local.worker_app_start_command
+  docker_image      = var.app_docker_image
+  health_check_type = "process"
+  instances         = var.web_app_instances
+  memory            = var.web_app_memory
+  space             = data.cloudfoundry_space.space.id
+  stopped           = var.app_stopped
+  strategy          = var.web_app_deployment_strategy
```

Comment:
I suggest to remove it to use the default (not blue-green). It doesn't matter to have zero downtime for workers, it's simpler and the deployment is faster.

Reply:
done...cheers.